plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/procedure/ExpireSnapshotsTableProcedure.java

```diff
@@ -36,6 +39,16 @@ public TableProcedureMetadata get()
                                 "retention_threshold",
                                 "Only snapshots older than threshold should be removed",
                                 Duration.valueOf("7d"),
+                                false),
+                        integerProperty(
+                                "retain_last",
```

Comment:
I used the parameter name which is the same as Spark here, but `min_snapshots` or something might be better. Other suggestions are welcome.

Reply:
`retain_last` sounds consistent with `retention_threshold` - I'd vote for that option